lambdas/raster_analysis/src/lambda_function.py

```diff
@@ -34,15 +34,18 @@ def handler(event, context):
         data_environment = DataEnvironment(layers=event["environment"])
         query = Query(event["query"], data_environment)
 
+        
         data_cube = DataCube(geom_tile.geom, geom_tile.tile, query)
+
         query_executor = QueryExecutor(query, data_cube)
         results: DataFrame = query_executor.execute()
 
         LOGGER.debug(f"Ran analysis with results: {results}")
-        results_store.save_result(results, context.aws_request_id)
+        results_store.save_result(results, event["tile_result_id"])
```

Comment:
minor: perhaps call this `cache_id` or something, I was little confused what this was until I look at how it was used. You also have all the information to call _get_cache_key in these lambdas, so you could just send the tile and query and get the cache key again in each lambda